src/App.vue

```diff
@@ -15,11 +18,14 @@ import auth from './auth/auth'
 // Components import
 import Login from './components/Login.vue'
 import Dashboard from './components/Dashboard.vue'
+import MobileAppNavigator from './components/mobile/MobileAppNavigator.vue'
```

Comment:
So far that's ok, but I am wondering if in future releases it could be fine to avoid the entire component loading in case of Desktop application. I mean I'd do something like:

if it is desktop, load the Dashboard vue component, otherwise load the MobileAppNavigator.

Reply:
You are right. We need a mechanism for lazy importing of components and a way to detect the platform not based on Onsen UI.